modules/apps/jaggery/mdm/src/mdm/modules/device.js

```diff
@@ -317,13 +317,19 @@ deviceModule = function () {
      @Updated
      */
     publicMethods.getLicense = function (deviceType, languageCode) {
-        return "";
-        //var url = mdmProps["httpsURL"] + "/mdm-admin/license/" + deviceType + "/" + languageCode;
-        //serviceInvokers.XMLHttp.get(url, function (responsePayload) {
-        //    return responsePayload.responseContent;
-        //}, function (responsePayload) {
-        //    return null;
-        //});
+        var url;
+        var license;
+        if(deviceType != null && deviceType != undefined && deviceType == "windows"){
+            url = mdmProps["httpURL"] + "/mdm-windows-agent/services/device/license";
+        } else{
```

Comment:
it has already fixed in https://github.com/wso2/product-mdm/blob/master/modules/apps/jaggery/emm/src/emm/modules/device.js